contentcuration/contentcuration/frontend/channelEdit/views/ResourcePanel.vue

```diff
@@ -196,6 +244,16 @@
       isResource() {
         return !this.isTopic && !this.isExercise;
       },
+      isImported() {
+        return this.node.original_channel_id !== this.channelId;
+      },
+      importedChannelLink() {
+        // TODO: Eventually, update with this.node.original_source_node_id for correct path
+        let channelId = this.node.original_channel_id;
+        let parentId = this.node.original_parent_id;
+        let originalNodeId = this.node.original_node_id;
```

Comment:
[many people prefer](https://softwareengineering.stackexchange.com/questions/278652/how-much-should-i-be-using-let-vs-const-in-es6) to use `const` instead of `let` for variables that are not intended to be reassigned